shacl12-profiling/index.html

```diff
@@ -524,12 +524,27 @@ <h3>Document Conventions</h3>
 
 <section id="packaging">
   <h2>Packaging SHACL</h2>
+  <p>
+    Packaging of SHACL is the presentation of SHACL elements - shapes, rules, etc. - in relation to container objects.
+  </p>
   <section id="packaging-motivation">
     <h3>Motivation</h3>
+    <div style="max-height:50px; overflow:hidden;">
+      <div class="issue" data-number="622" ></div>
+    </div>
+    <div style="max-height:50px; overflow:hidden;">
+      <div class="issue" data-number="623" ></div>
+    </div>
     <p>
-      For the efficient distribution and reuse of SHACL elements, we recommend packaging conventions. We make recommendations based on common SHACL packaging practice observed since the first release of SHACL in 2017 and follow generic Semantic Web definitional resource packaging practice too. However, achieve the level consistency required to allow for some of the aims of the following sections of this document to be met, we introduce some minor additions, such as the directive to use a particular part indicator predicate for the ontology &rarr; shapes relationship.
+      To enable the identification of profiles of graphs defined by collections of SHACL shapes and for the organisation of profile definitions and their elements, we recommend two SHACL elements packaging conventions. These based on, and only very slight extensions to, common SHACL packaging practice observed since the first release of SHACL in 2017. This also follows generic Semantic Web definitional resource packaging practice too.
```

Comment:
Too many long strings of nouns used to make compound nouns without punctuation. The reshuffling below should be substantially clearer to all readers, especially those who are new to the subject and/or using assistive technologies to get around visual and other issues.
```suggestion
      To enable the identification of profiles of graphs defined by collections of SHACL shapes and for the organisation of profile definitions and their elements, we recommend two packaging conventions for SHACL elements. These are based on, and are only very slight extensions to, common SHACL packaging practices observed since the first release of SHACL in 2017. These also follow generic packaging practices for definitional resources on the Semantic Web.
```